tutorials/3_Introduction-to-the-GUI.ipynb

```diff
@@ -51,13 +49,11 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
-   "metadata": {
-    "collapsed": false
-   },
+   "execution_count": null,
+   "metadata": {},
    "outputs": [],
    "source": [
-    "%matplotlib inline"
+    "%matplotlib notebook"
```

Comment:
@RemiLehe can you update this in all `.ipynb` files inside `tutorials/` please for consistency?

Reply:
I agree this would be more consistent, but it would complicate the other notebooks I think.

The reason is that many of the tutorials feature plots, which are shown just below the command that generated them. If I used `%matplotlib notebook`, this would no longer be the case, unless I add a `plt.figure(<new_number>)` before each command. However, I feel that this would make the notebook less readable.

Thus, would you be okay with keeping `inline` in these other notebooks.

Reply:
I understand, but wouldn't explicit figures instead of implicit figure assignment make sense in general? I see many people using the slider or some plots from the other APIs and own-read data together in the same notebook and using `%matplotlib notebook` as the default. This often leads to very confusing re-plots in figures they did not expect (because we don't show them the concept of figures).